dev/com.ibm.ws.st.core/src/com/ibm/ws/st/core/internal/config/validation/ConfigurationValidator.java

```diff
@@ -58,6 +61,45 @@ public ValidationResult validate(IResource resource, int kind, ValidationState s
             Trace.logError("Error removing markers", e);
         }
 
+        // For defined runtimes, do not validate any server config files in a templates folder inside the runtime folder
+        IPath fileLocation = file.getLocation(); // Full system path of the file
+        IPath fileFullPath = file.getFullPath(); // Used to ensure that templates is a subfolder of the runtime
+        WebSphereRuntime[] webSphereRuntimes = WebSphereUtil.getWebSphereRuntimes();
+        int length = webSphereRuntimes.length;
+        for (int i = 0; i < length; i++) {
+            WebSphereRuntime runtime = webSphereRuntimes[i];
+            IPath runtimeLocation = runtime.getRuntimeLocation();
+            if (runtimeLocation.isPrefixOf(fileLocation) && fileFullPath.toString().contains("templates")) { //$NON-NLS-1$
```

Comment:
To be safe I think this should check for runtimeLocation.append("templates").isPrefixOf(fileLocation) since the default usr directory is within the runtime.  The second check is not needed then.  And this matches better what you are doing below.

Reply:
Good catch.  It's a more refined check.  So it reduced down to this:

            if (runtimeLocation.append("templates").isPrefixOf(fileLocation)) { //$NON-NLS-1$
                return EMPTY_RESULT;
            }

Reply:
Changes made